src/utils/customFetch.ts

```diff
@@ -0,0 +1,15 @@
+import { HttpVerbs } from "src/@types/HttpVerbs";
+
+export function customFetch(
+  endpoint: string,
+  method: HttpVerbs,
+  body: Record<string, unknown>,
```

Comment:
Following Anthony's advices, I'm using Record type instead of Body

Reply:
My bad, actually it is `Record<string, string | number>` 🙇

Reply:
OK fixed!

Reply:
💪